main.tf

```diff
@@ -33,6 +33,7 @@ module "orca" {
   orca_default_bucket       = var.orca_default_bucket
   
   ## OPTIONAL
+  api_vpc_endpoint_id                                  = var.api_vpc_endpoint_id
```

Comment:
May want to call this just `vpc_endpoint_id` to remove confusion that this is the API Gateway id when in reality it is the NGAP VPC endpoint ID.

Reply:
changed to vpc_endpoint_id